pkg/extract/extract.go

```diff
@@ -117,9 +119,18 @@ func untarFile(tarReader *tar.Reader, header *tar.Header, path string) error {
 	}
 	defer file.Close()
 
+	var reader io.Reader
+	reader = tarReader
+	if isatty.IsTerminal(os.Stdout.Fd()) && header.FileInfo().Size() >= 1_000_000_000 {
```

Comment:
I'd start even at 100MB or so.

Reply:
done. In crcbundle, only the qcow2 file is larger than 100MB, so still only one progress bar.